target/linux/ramips/dts/rt5350_asiarf_awm002-evb-4m.dts

```diff
@@ -10,7 +10,7 @@
 &spi0 {
 	status = "okay";
 
-	m25p80: m25p80@0 {
+	m25p80: flash@0 {
```

Comment:
That label seems to be unnecessary. Maybe drop that as a "While at it" when refreshing patches after my merge (same for the next change on 8m variant).

Reply:
Nice catch. Removed now.